### PR TITLE
BZ1964586 Added notes in Deploying the Cluster Section

### PR DESCRIPTION
**Pull request by Wiharris**
**BZ1964586**
Added note not to exceed the maximum character limit for datastore and cluster name.

**Version(s):**
Versions 4.6 and beyond

**Issue:**
https://bugzilla.redhat.com/show_bug.cgi?id=1964586

**Link to docs preview:**
https://wiharris.github.io/openshift-docs/BZ1964586/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-launching-installer_installing-vsphere-installer-provisioned

Additional information:
**QE:** @duanwei33

### CODE REVIEW

**Comment by duanwei33** [comment]
@Wiharris Sorry I missed the tag ...
From the BZ description, I think we should mention the combined string doesn't exceed 60 characters? 
`And hence we should warn users from choosing long names for datastore and cluster name and recommend that users should not pick datastore and cluster-name in such a manner that their combined string length goes beyond 60 characters. `

Also for  "[] and /" limitation you mentioned, is this for OCP side? I mean vSphere allowed this but Openshift doesn't? If in this case, I think we should not choose such datastore?

**Comment by Wiharris** [comment]
> @Wiharris Sorry I missed the tag ... From the BZ description, I think we should mention the combined string doesn't exceed 60 characters? `And hence we should warn users from choosing long names for datastore and cluster name and recommend that users should not pick datastore and cluster-name in such a manner that their combined string length goes beyond 60 characters. `
> 
> Also for "[] and /" limitation you mentioned, is this for OCP side? I mean vSphere allowed this but Openshift doesn't? If in this case, I think we should not choose such datastore?

@duanwei33 I apologize for getting back so late! I checked and it is allowed by vSphere and OCP. I have removed the [] and /" references and updated the PR! :)

**Comment by Wiharris** [comment]
Hi @duanwei33 

> @Wiharris as mentioned in previous comments, could you help check: (And sorry miss the notification again!)
> 
> ```
> From the BZ description, I think we should mention the combined string doesn't exceed 60 characters?
> And hence we should warn users from choosing long names for datastore and cluster name and recommend that users should not pick datastore and cluster-name in such a manner that their combined string length goes beyond 60 characters.
> ```

@duanwei33 Done! :) I have updated the PR to include the above statement.

**Comment by duanwei33** [comment]
Thanks for the update.
/lgtm

File: modules/installation-launching-installer.adoc [hunk 1]
```diff
@@ -303,6 +303,11 @@ ifdef::vsphere,vmc[]
 The installation program connects to your vCenter instance.
 .. Select the datacenter in your vCenter instance to connect to.
 .. Select the default vCenter datastore to use.
++
+[NOTE]
+====
+DataStore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking DataStore and cluster names.
```
Comment (GroceryBoyJr) [inline modules/installation-launching-installer.adoc:309]: @Wiharris the "s" in Datastore is not capitalized in vSphere's documentation. Example usage: https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-3CC7078E-9C30-402C-B2E1-2542BEE67E8F.html

File: modules/installation-launching-installer.adoc [hunk 2]
```diff
@@ -312,6 +317,11 @@ endif::vsphere,vmc[]
 .. Enter a descriptive name for your cluster.
 ifdef::vsphere,vmc[]
 The cluster name must be the same one that you used in the DNS records that you configured.
++
+[NOTE]
+====
+DataStore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking DataStore and cluster names.
```
Comment (GroceryBoyJr) [inline modules/installation-launching-installer.adoc:323]: @Wiharris same as above.

**Review by GroceryBoyJr — COMMENTED** [review]
@Wiharris only one tiny little nit, sir.

File: modules/installation-launching-installer.adoc (hunk 2 above)
Comment (Wiharris) [reply modules/installation-launching-installer.adoc]: Done! :)

File: modules/installation-launching-installer.adoc (hunk 1 above)
Comment (Wiharris) [reply modules/installation-launching-installer.adoc]: Done! :)

File: modules/installation-launching-installer.adoc [hunk 3]
```diff
@@ -303,6 +303,11 @@ ifdef::vsphere,vmc[]
 The installation program connects to your vCenter instance.
 .. Select the datacenter in your vCenter instance to connect to.
 .. Select the default vCenter datastore to use.
++
+[NOTE]
+====
+Datastore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking datastore and cluster names.
```
Comment (nalhadef) [inline modules/installation-launching-installer.adoc:309]: Is the text "when picking datastore and cluster names" necessary?

File: modules/installation-launching-installer.adoc [hunk 4]
```diff
@@ -312,6 +317,11 @@ endif::vsphere,vmc[]
 .. Enter a descriptive name for your cluster.
 ifdef::vsphere,vmc[]
 The cluster name must be the same one that you used in the DNS records that you configured.
++
+[NOTE]
+====
+Datastore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking datastore and cluster names.
```
Comment (nalhadef) [inline modules/installation-launching-installer.adoc:323]: See previous.

File: modules/installation-launching-installer.adoc (hunk 3 above)
Comment (Wiharris) [reply modules/installation-launching-installer.adoc]: Thanks, @nalhadef and I agree! Adding "when picking datastore and cluster names" is not necessary as it is mentioned in the beginning part of the sentence. 

I have incorporated the above suggested changes! :)

File: modules/installation-launching-installer.adoc (hunk 4 above)
Comment (Wiharris) [reply modules/installation-launching-installer.adoc]: I have incorporated the above suggested changes! :)

**Pull request merged**